app/services/course_service.py

```diff
@@ -39,10 +39,11 @@ async def create_course(self, name: str, start_at: datetime = None, end_at: date
             raise CourseAlreadyExistsException()
         except NoCourseExistsException:
             pass
-
+        
+        print("CREATING COURSE")
```

Comment:
Did you mean to keep all the print statements in this file? If so shouldn't it be done by the logger instead of print()?

Reply:
I'll change these to logger debugs. I think they're useful to leave in the code for now at least in case any bugs with syncing crop up again.

Reply:
On second thought, I had asked Harit to work on getting the logger functional outside of app context. I'm not sure what he's done with that, or if we're just supposed to use module-level loggers or what. These should stay prints until then, since downsync runs outside of the app context (i.e. we downsync right before we start the uvicorn server), so trying to access the app logger would cause errors.